_posts/2018-08-17-snippets-brotli.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Using Brotli compression to reduce CDN costs"
+author: Giorgos Logiotatidis
+author_url: https://giorgos.sealabs.net
+excerpt_separator: <!--more-->
+---
+
+The Snippets Service allows Mozilla to communicate with Firefox users directly
```

Comment:
It might be nice to have a link from `Snippets Service` to the snippets repo

Reply:
"on their homepage". I think it is more accurate to write "on their new tab page"

Reply:
Done